src/Core/Components/Link/FluentLink.razor

```diff
@@ -0,0 +1,27 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentComponentBase
+
+<fluent-link href="@Href"
+			 hreflang="@HrefLang"
+			 referrerpolicy="@ReferrerPolicy.ToAttributeValue()"
+			 rel="@Rel.ToAttributeValue()"
+			 type="@LinkType"
+			 target="@Target.ToAttributeValue()"
+			 appearance="@Appearance.ToAttributeValue(returnEmptyAsNull: true)"
+			 inline="@Inline"
+			 id="@Id"
+			 class="@ClassValue"
+			 style="@StyleValue"
+			 @onclick="@OnClickHandlerAsync"
+			 @attributes="AdditionalAttributes">
+	@if (IconStart is not null)
+	{
+		<FluentIcon Value="@IconStart" Width="16" Margin="mr-1" />
```

Comment:
You should use `me-1` (End) because in LTR direction, the Left and Right are inverted. 

Same for `ml-1` => `ms-1`